src/minibook/plugins.py

```
@@ -231,12 +231,98 @@ def generate(
+
+    name = "pdf"
+    extension = ".pdf"
+    description = "Generate PDF output"
+
+    def generate(
+        self,
+        title: str,
+        links: list[tuple[str, str]],
+        subtitle: str | None = None,
+        output_file: str | Path = "links.pdf",
+        **kwargs,
```

Ruff (0.14.11)

Unused method argument: `kwargs`

(ARG002)